index.rst

```diff
@@ -50,6 +50,13 @@ For a complete list of our projects, check out `Open Data Kit on Github <https:/
 
   form-widgets
   javarosa
+
+.. toctree::
+  :hidden:
+  :maxdepth: 2
+  :caption: Form Design
+
+  intents
```

Comment:
intents should be added to the existing form-design toctree, not put in a new one